src/style.css

```diff
@@ -0,0 +1,191 @@
+:root {
+  --color-white: #fff;
+  --color-grey: #616070;
+  --color-black: #060b35;
+  --color-blue: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  width: 100%;
+  height: 100%;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+}
+
+h3,
+p {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+  color: var(--color-black);
+}
+
+/* card */
+.card {
+  position: relative;
+  display: block;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: var(--color-white);
+}
+
+.card__img {
+  position: relative;
+  width: 160px;
+  height: 0;
+  padding-bottom: 134px;
+  background-color: var(--color-grey);
+  margin: 0 auto 40px;
+}
+
+.card__img img {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  object-position: center;
+}
+
+.card__head {
+  position: relative;
+  width: 100%;
+  margin-bottom: 16px;
+}
+
+.card__title {
+  position: relative;
+  width: 100%;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--color-black);
+  margin-bottom: 4px;
+}
+
+.card__code {
+  position: relative;
+  width: 100%;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-grey);
+}
+
+.card__stars,
+.card__price {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  margin-bottom: 24px;
+}
+
+.card__price {
+  margin-bottom: 16px;
+}
+
+.card__stars p {
+  position: relative;
+  top: 1px;
```

Comment:
1) top: 1px; is redundant
2) use styles fro stars from a previous task
```suggestion
```

Reply:
Not fixed